great_expectations/metrics/metric_name.py

```diff
@@ -0,0 +1,11 @@
+from enum import Enum
+
+
+class DomainName(str, Enum):
+    BATCH = "table"
+    COLUMN_VALUES = "column_values"
+
+
+class MetricNameSuffix(str, Enum):
+    CONDITION = "condition"
```

Comment:
I added this because `condition` gets special treatment. Hopefully we won't find any more special cases like that and we can pare this file down.